javascript/javascript3/week1/shravani.json

```diff
@@ -0,0 +1,23 @@
+[
+    {
+        "song": "Jaamu Raatiri",
+        "Singer" : "Bala subrahmanyam",
+        "movie": "kshana kshanam",
+        "rating":{
+           "good":true,
+           "bad":false
+
+        }
+    },
+    {
+        "song":"leharaye",
+        "singer":"SidSreeRam",
+        "movie": "Most Eligible Bachelor"
+    },
+    {
+        "song":"emaindo",
+        "Singer":"manu",
+        "movie":"balupu"
+
```

Comment:
This is Hilarious. I liked it though. 😁

Reply:
thank you divya :) i did changes.... as review .....